ext/fg/js/content-script-main.js

```diff
@@ -47,7 +48,14 @@ async function createIframePopupProxy(frameOffsetForwarder, setDisabled) {
 }
 
 async function getOrCreatePopup(depth) {
-    const popupFactory = new PopupFactory();
+    const {frameId} = await apiFrameInformationGet();
+    if (typeof frameId !== 'number') {
+        const error = new Error('Failed to get frameId');
+        yomichan.logError(error);
+        throw error;
+    }
```

Comment:
When will this happen?

Reply:
Hopefully never in practice, but who knows with all the upcoming web extension changes. Could also happen if we tried to use the script in the browser action popup window or on the background page, which shouldn't be done.  The idea was just to provide a noticeable error in case something unexpected ever does break.